ginga/rv/plugins/Catalogs.py

```diff
@@ -1196,11 +1236,15 @@ def _unselect_tv(self, star, fromtable=False):
     def select_star_cb(self, widget, res_dict):
         """This method is called when the user selects a star from the table.
         """
-        key = list(res_dict.keys())[0]
-        idx = int(key)
-        star = self.starlist[idx]
-        if not self._select_flag:
-            self.mark_selection(star, fromtable=True)
```

Comment:
I would suggest `keys` instead of `key`, since we are making a list here and not just trying to select the first one.